### PR TITLE
Fixes #20766: Document the rudder-lang and technique editor incompatibilities

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/20766

### CODE REVIEW

**Review by peckpeck — APPROVED** [review]

File: language/docs/usage.adoc [hunk 1]
```diff
@@ -326,6 +326,59 @@ If you decided to go with the `--json-output` option, it means output will consi
...
+
+==== Example
+
+Let's write a very simple technique in rudder language:
+
+.technique.rd
+[source, language]
+----
+@format=0
+@name="Deploy vim" // <1>
```
Comment (ncharles) [inline language/docs/usage.adoc:351]: // is not recognised as a comment character by rudderc, so we can't copy paste the example

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**